trino/auth.py

```diff
@@ -156,9 +160,112 @@ def __eq__(self, other):
         return self.token == other.token
 
 
-def handle_redirect_auth_url(auth_url):
-    print("Open the following URL in browser for the external authentication:")
-    print(auth_url)
+class RedirectHandler(metaclass=abc.ABCMeta):
+    """
+    Abstract class for Oauth redirect handlers.
```

Comment:
nit: improve docstrings here and for classes below, like `OAuth`, remove extra spaces, keep dots etc.